src/main/java/dk/cngroup/lentils/controller/ClientController.java

```diff
@@ -0,0 +1,101 @@
+package dk.cngroup.lentils.controller;
+
+import dk.cngroup.lentils.entity.Cypher;
+import dk.cngroup.lentils.entity.CypherStatus;
+import dk.cngroup.lentils.service.CypherService;
+import dk.cngroup.lentils.service.HintService;
+import dk.cngroup.lentils.service.HintTakenService;
+import dk.cngroup.lentils.service.StatusService;
+import dk.cngroup.lentils.service.TeamService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+@RequestMapping("/client")
+public class ClientController {
+
+    private static final String CLIENT_VIEW_CYPHER_LIST = "client/list";
+    private static final String CLIENT_VIEW_CYPHER_DESCRIPTION = "client/detail";
+    private static final String REDIRECT_CLIENT_VIEW_CYPHER_LIST = "redirect:/client/list";
```

Comment:
REDITECT_TO_CLIENT_LIST (to view tam byt nema, pro5oze tam proviha redirect, ktery prejde do controller metody s tou url nikoli do view. Vytvori se novy request.